src/Services/FaraImporter.php

```diff
@@ -54,6 +69,18 @@ public function import(string $file)
         return $records;
     }
 
+    // Remove column names as specified by the columnFilter array without re-
+    // indexing the returned array.
+    protected function getFilteredColumns(string $tableName, array $columns): array
+    {
+        if (!array_key_exists($tableName, $this->columnFilter)) {
+            // No column filter available for this table.
+            return $columns;
+        }
+        $filter = $this->columnFilter[$tableName];
+        return array_filter($columns, fn($col) => !in_array($col, $filter));
```

Comment:
If we rename the filter variable, it is slightly clearer what is happening :)

```suggestion
        $ignoreCols = $this->columnFilter[$tableName];
        return array_filter($columns, fn($col) => !in_array($col, $ignoreCols));
```